index.html

```diff
@@ -0,0 +1,157 @@
+    <link href="https://fonts.googleapis.com/css2?family=Noto+Serif:wght@700&family=Open+Sans:wght@300&display=swap" rel="stylesheet">
+  </head>
+
+  <body>
+    <header>
+      <div class="my-cart-container">
+        <img class="my-cart-box" src="images/cart-icon.svg" alt="empty">
+        <p class="cart-total">$1234.56</p>
+        <a class="checkout-button" href="checkout.html">Checkout</a>
+      </div>
+
+      <div class="main-logo">
+        <img class="logo" src="images/adoptadog-logo.svg" alt="Adoptadog Logo">
+        <h1>Seattle's premiere dog adoption agency</h1>
+      </div>
+
+      <nav>
+        <ul>
+          <li><a href="index.html">Home</a></li>
+          <li>/</li>
+          <li><a href="dogs.html">Dogs</a></li>
+          <li>/</li>
+          <li><a href="blog.html">Blog</a></li>
+        </ul>
+      </nav>
+    </header>
+
+    <aside>
+      <div>
+        <h2>From our Blog</h2>
+      </div>
+      <div class="blog-preview-img">
+        <img src="images/blog-1.jpg" alt="Human sitting with dog on ledge of a cliff">
+      </div>
+      <div class="blog-preview-txt">
+        <h3>Traveling With Your Dog</h3>
+        <p class="blog-preview-text">Iciustiam, et ad quiaes cum que excea sinciendi qui offic temquas doluptios dolupta tiandebis nem ut.</p>
+        <a class="read-more" href="blog.html"> Read More >></a>
+      </div>
+
+      <div class="blog-preview-img">
+        <img src="images/blog-2.jpg" alt="Multiple dogs sitting in a park on leashes">
+      </div>
+      <div class="blog-preview-txt">
+        <h3>How To Walk Multiple Dogs</h3>
+        <p class="blog-preview-text">Iciustiam, et ad quiaes cum que excea sinciendi qui offic temquas doluptios dolupta tiandebis nem ut.</p>
+        <a class="read-more" href="blog.html"> Read More >></a>
+      </div>
+
+      <div class="blog-preview-img">
+        <img src="images/blog-3.jpg" alt="Human asking dog to sit and dog sitting in the sunset">
+      </div>
+      <div class="blog-preview-txt">
+        <h3>Teach Your Dog To Fetch!</h3>
+        <p class="blog-preview-text">Iciustiam, et ad quiaes cum que excea sinciendi qui offic temquas doluptios dolupta tiandebis nem ut.</p>
+        <a class="read-more" href="blog.html"> Read More >></a>
+      </div>
+    </aside>
+
+    <div class="non-header-body">
+      <div class="index-container">
+        <div class="index-box-img">
+          <img src="images/home-intro.jpg" alt="Boy hugging dog">
+        </div>
+
+        <div class="index-box-text">
+          <h2>Adopt A Dog</h2>
+          <p class="index-box-text-p">Iduciendisite quo magnatem iuntum quid quaest ea am, tenderumet adis dolenem quidustrum fuga. Faceaquae estioria derum recuptatur, cum volore, undipsa doloreium hillupta aut es ut alitatuscit ommossum haritatur arum qui officae videbiti corporeium faccull oribus es quidignis ipietus explam sus am</p>
+        </div>
+      </div>
+
+      <div class="dogs-container">
```

Comment:
One thing I would suggest here is look at making your cards slightly bigger on mobile. They seem to get pushed pretty small on mobile. This would be a great use of media queries!

Reply:
Thanks, Tyler! I tried a query and it worked for the dogs page, but did not replicate to the home page.